run.py

```diff
@@ -0,0 +1,65 @@
+import os
+from src.model import MLProphet
+from src.data_extract import yahoo
+from sklearn.metrics import r2_score
+from datetime import datetime
+import datetime as dt
+
+
+TICKER = os.environ.get("TICKER", "tsla")
+LABEL = os.environ.get("LABEL", "Open")
+# CURRENT_DATE = os.environ.get("CURRENT_DATE", datetime.today().strftime("%Y-%m-%d"))
+CURRENT_DATE = "2022-04-01"
+WINDOW = os.environ.get("WINDOW", 1)
+
+FEATURES = 'Open,High,Low,Close,Adj Close,Volume'.split(",")
+
+
+if __name__ == '__main__':
+    # TODO: Adjust dates based on weekdays.
+    start_date = (datetime.strptime(CURRENT_DATE, "%Y-%m-%d").date() - dt.timedelta(365)).strftime("%Y-%m-%d")
+
+    train_start = (datetime.strptime(CURRENT_DATE, "%Y-%m-%d").date() - dt.timedelta(365)).strftime("%Y-%m-%d")
+    train_end = CURRENT_DATE
+    pred_start = (datetime.strptime(CURRENT_DATE, "%Y-%m-%d").date() - dt.timedelta(1)).strftime("%Y-%m-%d")
+    pred_end = (datetime.strptime(CURRENT_DATE, "%Y-%m-%d").date() + dt.timedelta(WINDOW-1)).strftime("%Y-%m-%d")
+
+    yh_train = yahoo(TICKER, train_start, train_end)
+    data_yh_train = yh_train.collect_data()
+
+    yh_pred = yahoo(TICKER, pred_start, pred_end)
```

Comment:
looks like the current run file is for model offline evaluation, should we include the in-future prediction here or in a separate file?